docset/windows/addsadministration/Remove-ADGroupMember.md

```diff
@@ -1,7 +1,7 @@
 ---
 ms.mktglfcycl: manage
 ms.sitesec: library
-ms.author: v-kaunu
+ms.author: jaiello
```

Comment:
Same comment as #2229 about changing author metadata

Reply:
The original vendor account was not valid anymore. I tried different users I saw as contributing to this area of DOCS.
You can use the best fit for your purposes...

Reply:
Let's try to put v-kaunu ( @kateyanne ) again. Thank you @HerbertMauerer